vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java

```diff
@@ -75,7 +73,7 @@ private ResourceBundle extractUnsafeBundle(Object bundle, Locale locale) {
 			String baseName = firstNonNull((String) bundle, DEFAULT_BUNDLE_NAME);
 
 			try {
-				return ResourceBundle.getBundle(baseName, locale);
+				return ResourceBundle.getBundle(baseName, locale, new UTF8Control());
```

Comment:
instead of changing the expected/default behaviour, 
what do you think about creating an CDI alternative for utf8 bundle producer?
so users can enable it via beans.xml configuration

http://docs.oracle.com/javaee/6/tutorial/doc/gjsdf.html